### DISCUSSION TITLE
Lookups

### DISCUSSION THREAD

**Discussion by W1DGL-1** [foldynl/QLog#423] [General]
Unsure what the "yellow marked fields are invalid" means when doing a lookup (QRZ, in my case) to edit a QSO/call.
Also would like to know how to change the clock to display 24-hr time instead of 12-hr, even if it is UTC.

**Comment by foldynl** [comment]
yellow marked fields indicate certain inconsistencies in the data. The best thing to do is to move your mouse to the yellow field and  Field's Tooltip shows what QLog expects. If it expects something that is not true, you can ignore the warning.

>>>> Also would like to know how to change the clock to display 24-hr time instead of 12-hr, even if it is UTC.

Please, see [QLog Wiki](https://github.com/foldynl/QLog/wiki/Multi-language-Support#date--time-format). QLog uses the settings from LC_* variables (in Linux).

**Reply by W1DGL-1** [reply]
I tried that - when I hover over the yellow fields, there's no tooltip there to show me what the field 'expects' to be there, at least not on the Linux version.  This only happens when I edit and do a lookup (for an .adi file I uploaded, for example), so I just ignore it and save it anyhow.

**Reply by foldynl** [reply]
Please, could you send a screenshot ? My screenshot below.

![warning](https://github.com/user-attachments/assets/f6aa998c-fbff-4c49-8375-68bec79b22df)

**Reply by W1DGL-1** [reply]
Duh (on me).  I was thinking that since the CHANGES were yellow, that's what the notice meant; it's the HEADING that turns yellow if the info entered is invalid for the call. (again, DUH, me!)

**Comment by W1DGL-1** [comment]
![image](https://github.com/user-attachments/assets/c76fafc6-5078-4a60-9b94-2ca52b3104ba)
This is after doing a Lookup, but before saving it.

**Comment by foldynl** [comment]
QLog shows a warning because ITU zone for France is 27.
